pkg/generators/markers_test.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package generators_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/kube-openapi/pkg/generators"
+)
+
+func TestParseCommentTags(t *testing.T) {
```

Comment:
Unit tests are good. Probably would like to see more cases fleshed out. Also we want integration tests added to https://github.com/kubernetes/kube-openapi/blob/master/pkg/generators/openapi_test.go#L0-L1

Reply:
Also we have some good "integration" tests that show that the final generate openapi is good given the actual comments and stuff. They give a lot of confidence.